### PR TITLE
docs: add percona to list of Argo users

### PR DESCRIPTION
**Pull request by edithturn**
Hi, Argo CD team!

I am adding Percona to the list of users.
cc @terrytangyuan

### CODE REVIEW

**Review by terrytangyuan — COMMENTED** [review]
LGTM

File: USERS.md [hunk 1]
```diff
@@ -25,7 +25,7 @@ Currently, the following organizations are **officially** using Argo CD:
 1. [AppDirect](https://www.appdirect.com)
 1. [Arctiq Inc.](https://www.arctiq.ca)
 1. [ARZ Allgemeines Rechenzentrum GmbH](https://www.arz.at/)
-2. [Autodesk](https://www.autodesk.com)
+1. [Autodesk](https://www.autodesk.com)
```
Comment (terrytangyuan) [inline USERS.md:28]: Nice catch!

**Review by alexmt — APPROVED** [review]

File: USERS.md (hunk 1 above)
Comment (edithturn) [reply USERS.md:28]: Ohh. That is some else contribution. Maybe my mistake was not rebase it and I took that fix too

**Comment by terrytangyuan** [comment]
@edithturn You'll need to sign your commit to pass DCO check.

**Review by blakepettersson — APPROVED** [review]
:shipit:

**Comment by edithturn** [comment]
> @edithturn You'll need to sign your commit to pass DCO check.

I just did a --amend -S, to sign my commit
I hope it works

File: USERS.md [hunk 2]
```diff
@@ -321,3 +321,4 @@ Currently, the following organizations are **officially** using Argo CD:
 1. [ZDF](https://www.zdf.de/)
 1. [Zimpler](https://www.zimpler.com/)
 1. [ZOZO](https://corp.zozo.com/)
+1. [Percona](https://percona.com/)
```
Comment (blakepettersson) [inline USERS.md:324]: Could this be moved up in alphabetical order?

Comment (edithturn) [reply USERS.md:324]: Of course :)
I am trying to signoff those as well., thank you!

**Comment by edithturn** [comment]
> @edithturn that didn't seem to work.. and now you have more commits so you'll need to signoff those as well

Hi, @blakepettersson I think it is working now.
Appreciate your support.

**Pull request merged**